src/main/java/com/mixpanel/android/surveys/SurveyActivity.java

```diff
@@ -113,57 +120,36 @@ private void onCreateInAppNotification(Bundle savedInstanceState) {
             closeButtonWrapper.setLayoutParams(params);
         }
 
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
-            backgroundImage.setBackgroundColor(getResources().getColor(R.color.com_mixpanel_android_inapp_dark_translucent));
+        inAppImageView.showShadow(inApp.shouldFadeImage());
```

Comment:
the name `showShadow` is a bit confusing here. can we call it something like `setShouldShowShadow`?